src/main/java/com/onlinebookstore/dto/order/CreateOrderRequestDto.java

```diff
@@ -0,0 +1,10 @@
+package com.onlinebookstore.dto.order;
+
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+
+public record CreateOrderRequestDto(
+        @NotNull @Size(max = 255)
```

Comment:
```suggestion
        @NotBlank @Size(max = 255)
```

Reply:
Here the idea is to check only for NotNull, to let using of default user.shippingAddress, if shippingAddress not specified in request explicitly. Please, don't consider this as a mistake.